pkgs/by-name/wa/wakatime-cli/package.nix

```
@@ -8,23 +8,27 @@
 
 buildGoModule rec {
   pname = "wakatime-cli";
-  version = "1.102.3";
+  version = "1.112.1";
 
   src = fetchFromGitHub {
     owner = "wakatime";
     repo = "wakatime-cli";
-    rev = "v${version}";
-    hash = "sha256-RnEyrbdYP8BYQ9M+iwikTfYF8HRosR6vhHiQJvGNZ/s=";
+    tag = "v${version}";
+    hash = "sha256-J8hEQkTCaJhYYnuO8J2Tvgm/lWuPKHubKESb7WPzzkk=";
   };
 
-  vendorHash = "sha256-s9nCr55XR/0ZriweDOBRIDpmp+GUE85FWLJkWXqX5/U=";
+  vendorHash = "sha256-Zy54fGyTvLfrFEiPIcNtx25EqaIaYq46DYBpbZXfOFQ=";
 
   ldflags = [
     "-s"
     "-w"
     "-X github.com/wakatime/wakatime-cli/pkg/version.Version=${version}"
   ];
 
+  preCheck = ''
+    export HOME=$(mktemp -d)
+  '';
```

We have a [hook](https://github.com/NixOS/nixpkgs/blob/master/doc/stdenv/stdenv.chapter.md#writable-tmpdir-as-homesh-writable-tmpdir-as-homesh) doing this now.
```suggestion
  nativeCheckInputs = [
    writableTmpDirAsHomeHook
  ];
```

---

Presumably your nixpkgs is not new enough to include an important fix for that hook: https://github.com/NixOS/nixpkgs/pull/378927